packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubbleContent.base.tsx

```diff
@@ -89,7 +89,7 @@ export class TeachingBubbleContentBase extends React.Component<ITeachingBubblePr
 
       headerContent = (
         <div className={classNames.header}>
-          <HeaderWrapperAs role="heading" className={classNames.headline} id={ariaLabelledBy}>
+          <HeaderWrapperAs role="heading" aria-level={3} className={classNames.headline} id={ariaLabelledBy}>
```

Comment:
Do you know for sure that this will always be the correct level? cc @smhigley in case you have any thoughts on this.

Reply:
I was previouly want to cherry pick the change from 8.0 version. But as the 7.0 version package path is different, thus I directly created the PR and copy the fix from that. Any thought on if we shoud provide the interface for consumer to customize the level or provide other level for it?
https://github.com/microsoft/fluentui/commit/55babc27987f92716147367ba35338f88a4566be

Reply:
Oh okay, that's fine then (I thought I'd seen this change before)